### PR TITLE
Eliminate whitespace and commas from MongoDB match candidates (and resulting fingerprints).

### PR DESCRIPTION
**Pull request by eddynaka**
## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [x] `ReleaseHistory.md` updated for non-trivial changes
* [x] Added unit tests

### CODE REVIEW

**Comment by michaelcfanning** [comment]
Where is this documented in the release notes? Also, please note the update I made to this PR, let's try to be a bit more descriptive.

**Review by michaelcfanning — CHANGES_REQUESTED** [review]
:clock1:

**Comment by eddynaka** [comment]
I first open the PR and do a second commit adding the releasehistory to get the #. I will also update the releasehistory with that.

---
In reply to: [915524691](https://github.com/microsoft/sarif-pattern-matcher/pull/554#issuecomment-915524691) [](http://example.com/codeflow?ancestors=915524691,915524691)

**Comment by michaelcfanning** [comment]
Here's the title I chose: Eliminate whitespace and commas from MongoDB match candidates (and resulting fingerprints).

If we work together on good titles, these also serve as good candidates for the release notes documentation.

---
In reply to: [915524691](https://github.com/microsoft/sarif-pattern-matcher/pull/554#issuecomment-915524691) [](http://example.com/codeflow?ancestors=915524691,915524691)

File: Src/ReleaseHistory.md [hunk 1]
```diff
@@ -18,10 +18,13 @@
 
 - FPC: Improving RabbitMQ regex (removing new lines and spaces) from secret.
   [#548](https://github.com/microsoft/sarif-pattern-matcher/pull/548)
-- FPC: Normalizing regular expressions (`\s\n` got replaced by `\s`). Rules 
+- FPC: Normalizing regular expressions (`\s\n` got replaced by `\s`). Rules
   `SEC101/036.MySqlCredential`, `SEC101/037.SqlCredentials`,
   `SEC101/038.PostgreSqlCredentials` won't accept spaces in `id` and `secret`.
   [#550](https://github.com/microsoft/sarif-pattern-matcher/pull/550)
+- FPC: Improving MongoDB regular expression removing hosts that contain `\s` or
+  `,` characters.
```
Comment (michaelcfanning) [inline Src/ReleaseHistory.md]: 'Eliminate whitespace and characters from MongoDB host matches (and resulting fingerprints)'.

**Review by michaelcfanning — APPROVED** [review]
:shipit:

**Pull request merged**